### ISSUE TITLE
Issue with the selenium requirement

### ISSUE THREAD

**Issue by PaxJaromeMalues** [UnMars/Prolific-Joiner#23]
Hi there,

I made it through some of the requirements issues with my 3.12 installation on my own (ffmpeg not pathed, package upgrades required on wheel and playsound as well as manually retrieving blinker), but now I am hitting an issue that is deeper in the code which I am not able to fix.

apparently google did some changes to selenium and the CDM can now not be retireved from seleniumwire beyond version 114 (version in script appearing is 125).

apparently this can be fixed by restructuring the script to retrieve `webdriver` straight from 4.10 selenium instead. But I do not have the qualification to make out, where that is in your code and even less how to repair it.

**Comment by PaxJaromeMalues** [comment]
Oh apparently someone also came to this conclusion, but I am too inapt to implement it:
https://stackoverflow.com/a/76734893

**Comment by PaxJaromeMalues** [comment]
hello?

**Comment by usdbrl** [comment]
@PaxJaromeMalues 
 had a similar issue, the solution for now was use Python 3.11.
You you have a error related with "blinker", use "pip install blinker==1.7.0" to install the right version.

**Comment by PaxJaromeMalues** [comment]
@usdbrl Yep I got rid off the blinker stuff as written. 3.11 did sadly not work for me.
@UnMars Any chance of you throwing an update in over the next months?

**Comment by UnMars** [comment]
I don't have the problem on my side, i'm on python 3.10.12, it's fully working.
Also, selenium is still maintained so i don't really understand the issue here
